README.md

```
@@ -308,32 +308,33 @@ The Reclaim SDK offers several advanced options to customize your integration:
+   Set a custom URL to redirect users on a cancellation which aborts the verification process:
+  
+```javascript
+reclaimProofRequest.setCancelRedirectUrl("https://example.com/error-redirect");
+```
 
 5. **Custom Callback URL**:
    Set a custom callback URL for your app which allows you to receive proofs and status updates on your callback URL:
    Pass in `jsonProofResponse: true` to receive the proof in JSON format: By default, the proof is returned as a url encoded string.
 
-   ```javascript
    reclaimProofRequest.setAppCallbackUrl("https://example.com/callback", true);
```

markdownlint-cli2 (0.20.0)

[warning] Bare URL used

(MD034, no-bare-urls)